src/charm.py

```diff
@@ -22,14 +27,39 @@
 class KratosCharm(CharmBase):
     """Charmed Ory Kratos."""
 
+    admin_port = 4434
```

Comment:
Could we make these port variables uppercase on module level as they are constants?